src/openapi/openapi.json

```diff
@@ -2522,6 +2522,512 @@
         "x-router-controller": "cors"
       }
     },
+    "/api/data/inputs": {
+      "get": {
+        "tags": [
+          "data"
+        ],
+        "summary": "Get all input details",
+        "description": "Returns all input details",
+        "operationId": "inputFindAll",
+        "responses": {
+          "200": {
+            "description": "OK",
+            "headers": {
+              "Access-Control-Allow-Origin": {
+                "schema": {
+                  "type": "string"
+                }
+              },
+              "Access-Control-Allow-Methods": {
+                "schema": {
+                  "type": "string"
+                }
+              },
+              "Access-Control-Allow-Headers": {
+                "schema": {
+                  "type": "string"
+                }
+              }
+            },
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/Input"
+                }
+              }
+            }
+          },
+          "500": {
+            "description": "Bad request"
+          }
+        },
+        "security": [
+          {
+            "Cognito_Authorizer": ["openid"]
+          }
+        ],
+        "x-amazon-apigateway-integration": {
+          "uri": "${data_lambda_arn}",
```

Comment:
This is pointing to the data lambda where it should be pointed to the question lambda